transactions.md

```diff
@@ -78,6 +78,10 @@ fileName:transaction.json
 
 > The `authCode` property is present only for Mastercard and Visa transactions, and will mirror the `identifiers.mastercardAuthCode` or `identifiers.visaAuthCode` properties, depending on the issuing card for the transaction. The `approvalCode` property is present only for Amex transactions and will mirror the `identifier.amexApprovalCode` property.
 
+> The `identifiers.mastercardRefNumber` is present for Mastercard transactions. This property holds a globally unique number by the transaction date and is used by Fidel API to try to match a given clearing transaction with the corresponding authorisation transaction.
```

Comment:
Should we create different inner sections for fields? I think it would be more organized explaining the object and could grow more easily.

Reply:
I believe there is something planned to do that as @krisztina-nagy has been documenting [here](https://docs.google.com/document/d/1UCkX9B7Q2dmQlaaPdFvVrECMjaaDDxw3Z6nIAYiFXGk/edit). Not sure what are the plans to move this to our docs.